### DISCUSSION TITLE
PHP 8 support

### DISCUSSION THREAD

**Discussion by georak** [spatie/laravel-tags#336] [Q&A] [answered]
When you ll support php 8?

**Comment by freekmurze** [comment, marked as answer]
We already do: https://github.com/spatie/laravel-tags/blob/caedcd9636c4560eb2f44b0756b87a95f04b9d21/composer.json#L19

**Reply by georak** [reply]
Thank you! I was using an older version

**Reply by freekmurze** [reply]
![tumblr_n51256VmcZ1qlxnhco1_250](https://user-images.githubusercontent.com/483853/122045891-89025f80-cdde-11eb-9b09-1d8a3c30a6b4.gif)
